cmd/red-box/main.go

```diff
@@ -47,22 +45,27 @@ func main() {
 	var wg sync.WaitGroup
 	wg.Add(1)
 
-	srv := http.Server{Handler: router}
+	s := grpc.NewServer()
+
+	c, err := local.NewClient()
```

Comment:
Maybe this can be moved to `NewSlurmAPI`?
also I think `local` pakage is not needed and can be extracted in `slurm` directly